js/nixos-site.js

```diff
@@ -1,3 +1,25 @@
+$.fn.shuffleChildren = function() {
+  $.each(this.get(), function(index, el) {
+      var random = function(s = Date.now()) {
```

Comment:
What's wrong with just shuffling randomly on page load? Would be simpler. The general idea is to present companies without preferred ordering, so it shouldn't matter how exactly that is achieved.

Reply:
Depends if we want to have the same behavior as before. Before this PR, the page was updated every hour.
By seeding the PRNG with the current `hour` of the day, we make sure that the order will change every hour.
If you don't want that behavior and have a different order at each page load, I can restore the original version I did, I don't mind.

Reply:
Let @garbas take a look, he's ultimately responsible.

Reply:
The function is called with `(new Date).getHours()`, setting the seed to that.
This approach generates 24 permutations, one for each hour in a day.
But then, we only cycle through those same permutations every day.

More fundamentally, it might just be preferable to just drop this hourly 
change and just shuffle on each page load instead.

This remove the need to implement a custom sequence PRNG in JS,
which is bad for everyone's sanity.

Reply:
IMHO, I also thing this is the best option, see JSFiddle: https://jsfiddle.net/gb63joLy/1/